src/tracer.js

```diff
@@ -68,6 +68,20 @@ export default class Tracer {
         let binaryCodec = new BinaryCodec();
         this.registerInjector(opentracing.FORMAT_BINARY, binaryCodec);
         this.registerExtractor(opentracing.FORMAT_BINARY, binaryCodec);
+
+        this._setProcess();
+    }
+
+    _setProcess(): void {
+        let tags = [];
+        for (let key in this._tags) {
+            let value = this._tags[key];
+            if (this._tags.hasOwnProperty(key)) {
+                tags.push({ 'key': key, 'value': value });
+            }
+        }
```

Comment:
what's the purpose of this conversion? It's less critical than with log fields, but I'd rather have fewer lines of code, and this repackaging of the tags/fields seems unnecessary.